evap/static/ts/src/notebook.ts

```diff
@@ -1,11 +1,6 @@
 import "./translation.js";
 import { unwrap, assert, selectOrError } from "./utils.js";
 
-const NOTEBOOK_LOCALSTORAGE_KEY = "evap_notebook_open";
```

Comment:
For the localstorage key, we have two options:

1. The notebook assumes it is only ever loaded once into a page. Then, the key can be encapsulated within the .ts file, no other component needs to know about it.
2. The notebook might be loaded multiple times. We must prevent collisions then. One way to do this would be to have the calling code pass a unique id. The current code passes the generic term "evap_notebook_open", the notebook will then concatenate the ID (#notebook) in line 54 below. This is somewhat overcomplicated.

I'm fine with any of the two approaches, but I'd like it to be straightforward. If the calling code passes a localstorage key, just use that.